apps/server/src/modules/tool/controller/mapper/external-tool-response.mapper.spec.ts

```diff
@@ -193,4 +195,45 @@ describe('ExternalToolResponseMapper', () => {
 			});
 		});
 	});
+
+	describe('mapToConfigurationTemplateResponse', () => {
```

Comment:
Please use this https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/shared/domain/entity/submission.entity.spec.ts#L164